web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -63,8 +69,12 @@ Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/`.
 The file is a markdown file, you can use `web/pdeps/0001.md` as a reference
 for the expected format.
 
-The initial status of a PDEP will be `Status: Under discussion`. This will be changed
-to `Status: Accepted` when the PDEP is ready and have the approval of the core team.
+The initial status of a PDEP will be `Status: Draft`. Once it is ready for discussion, the author(s)
+change it to `Status: Under discussion`, and the following are notified: core and triage teams 
+(by tagging them on GitHub: `@pandas-core`, `@triage teams`) and the 
+[pandas-dev mailing list](mailto:pandas-dev@python.org). This will be changed to `Status: 
+Accepted` when the PDEP is ready and have the approval of the core team.
```

Comment:
Based on discussion in governance meeting on 3/15/2023, we agreed to remove this change from this PR, as it will be covered in a second PR related to the voting process  @noatamir